tests/e2e/files/rulebooks/actions/test_actions_sanity.yml

```diff
@@ -110,13 +112,27 @@
           module_args:
             msg: "Retracted fact in same ruleset, this should not be printed"
 
+    - name: Test multiple actions
+      condition: event.action == "multiple_actions"
+      actions:
+        - run_module:
+            name: ansible.builtin.debug
+            module_args:
+              msg: "Multiple action #1 triggered"
+        - debug:
```

Comment:
@Alex-Izquierdo should we also test if the same action is used more than once? I.e. two separate debug actions.

Reply:
It's a *sanity* test. That case is already applied in your PR: https://github.com/ansible/ansible-rulebook/pull/375/files#diff-eafffb1901d6d199cb5a7a9823449471157c108b5f418525ca8e271ed32c0faaR18